game-core/src/main/java/games/strategy/triplea/ui/logic/RouteCalculator.java

```diff
@@ -3,25 +3,28 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import java.awt.geom.AffineTransform;
 import java.awt.geom.Path2D;
 import java.awt.geom.Point2D;
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import javax.annotation.Nullable;
 
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 
-/**
- * Utility class to calculate a smooth route to draw on the map.
- */
+@AllArgsConstructor(access =  AccessLevel.PUBLIC)
 @Builder
 public class RouteCalculator {
 
-  public final boolean isInfiniteX;
   public final boolean isInfiniteY;
+  public final boolean isInfiniteX;
```

Comment:
The members were reversed

Reply:
Because this snapshot is still using `@AllArgsConstructor`, the member declaration order dictates the parameter order in the constructor.  There is one call to the constructor in `ImageScrollerSmallView` that still specifies `isInfiniteY` before `isInfiniteX`, so this field order is technically correct for the 1.9.0.0.10506 snapshot.

Before reordering the fields, the all-args constructor should probably be made private, and the remaining call site in `ImageScrollerSmallView` should be converted to builder usage.